stl/inc/functional

```diff
@@ -29,16 +29,12 @@ _STL_DISABLE_CLANG_WARNINGS
 #undef new
 
 _STD_BEGIN
-// STRUCT TEMPLATE plus
 // defined in <xstddef>
 
-// STRUCT TEMPLATE minus
 // defined in <xstddef>
 
-// STRUCT TEMPLATE multiplies
 // defined in <xstddef>

```

Comment:
Should these be replaced with a comment saying the others are defined elsewhere?